### PR TITLE
show planning table only for logged in and welcome for logged out

### PR DESCRIPTION
**Pull request by pflopez**
PLAN-348
PLAN-349

Updating the home page so the welcome message will only show for logged out users, and the planning area only shows up for logged in users.

I've added an extra link on the welcome text to log in (not present on figma) , similar to the current guest experience with login flag enabled. 

- Moving preview copy to separate component
- Keeping plan-table only for plan table stuff
- Adding `planning areas` component to manage plans
- Adding welcome component to show welcome text for guests with login feature flag on.

If login feature flag is off, we'll still show the planscape preview help text.

## with login feature flag off
Screen Shot 2023-09-04 at 09 26 36

### guest
Screen Shot 2023-09-04 at 09 24 59

### logged in
Screen Shot 2023-09-04 at 09 25 12

### CODE REVIEW

File: src/interface/src/app/home/home.component.html [hunk 1]
```diff
@@ -1,2 +1,4 @@
-<app-region-selection></app-region-selection>
-<app-plan-table></app-plan-table>
+<app-region-selection
+  *ngIf="(loggedIn$ | async) === false"
+></app-region-selection>
+<app-plan-table *ngIf="loggedIn$ | async"></app-plan-table>
```
Comment (sachiyuan) [inline src/interface/src/app/home/home.component.html:4]: I think the home/welcome text lives in this table (toggle between plan list and welcome text is based off of the login flag). If we're only showing the table for logged in users, has the welcome text moved somewhere else for not logged in users?

Comment (pflopez) [reply src/interface/src/app/home/home.component.html:4]: This part is the `<app-region-selection>`
Screen Shot 2023-09-01 at 14 23 35

Comment (sachiyuan) [reply src/interface/src/app/home/home.component.html:4]: Yeah, sorry if I misplaced the comment, I was talking about line 4: <app-plan-table *ngIf="loggedIn$ | async"></app-plan-table>

Comment (pflopez) [reply src/interface/src/app/home/home.component.html:4]: right but not sure i follow.. we have 2 "welcome to planscape preview" texts now. The one on region selection and the one above.
Do we need both? Is that the piece you mention is missing?

Comment (pflopez) [reply src/interface/src/app/home/home.component.html:4]: talked this out on a call and figured out what I was missing, thanks @sachiyuan will mark this as draft until is good to go

**Review by sachiyuan — APPROVED** [review]
LGTM

**Pull request merged**